### PR TITLE
Our Code Went To Cloud.

### PR DESCRIPTION
**Pull request by takkanm**
When a social coding environment comes to your workplace, how should an ordinary programmer feel and act?

### CODE REVIEW

**Comment by takkanm** [comment]
英語は後で追加します

**Comment by agrimm** [comment]
I wasn't able to find your email address, so I created a private gist with a minor edit of your English at https://gist.github.com/ae50a71cf99910c41c08 .

There were a few bits I didn't quite understand - I put some comments in square brackets.

**Comment by takkanm** [comment]
@agrimm thank you!!

File: takkanm-our-code-went-to-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+私は永和システムマネジメントという会社でRubyプログラマをやっています。
+
+弊社には、edgeなRubyモヒカンや、古きよきオブ厨が多数います。
+昨年から、会社ではGitHubが導入され、それまでプロジェクト内だけで共有されていたコードが、
+社内とはいえ様々な人の目につきやすい状況になり、様々なフィードバックを得やすい状況となりました。
+
+今後、仕事でもGitHub等のソーシャルコーディングが行なえる環境が普及していくと思います。
+自分の職場にソーシャルコーディングができる環境がやってきたとき、
+ふつうのプログラマがどのように立ち振るまうとよいのかということ、
+実体験を交わえ話します。
```
Comment (ursm) [inline takkanm-our-code-went-to-cloud/README.md]: 交え

Comment (takkanm) [inline takkanm-our-code-went-to-cloud/README.md]: thx

**Comment by adzuki34** [comment]
:thumbsup:

**Comment by snoozer05** [comment]
## English

Thank you for your proposal for Sapporo RubyKaigi2012.

Congratulations! We’re pleased to tell you that your proposal was accepted.

Please join the speaker mailing-list so that we can adjust the
timetable for your presentation. Please send an e-mail to
sapporo2012@rubykaigi.org by June 7 with the following format:
- Subject: sprk2012 speaker
- To: sapporo2012@rubykaigi.org
- Body: the URL of your pull request

## Japanese

この度はご応募いただきありがとうございました。

Sapporo RubyKaigi2012 発表に採択されたことをお知らせします。おめでとうございます。

タイムテーブルの調整などを行う必要がありスピーカー向けのメーリングリストに入っていただきたいので、
6/7 までに以下のフォーマットで sapporo2012@rubykaigi.org までメールにてご連絡ください。
- Subject: sprk2012 speaker
- To: sapporo2012@rubykaigi.org
- Body: pull requestのURL

**Pull request merged**